plugins/search/providers/elastic_search/services/ESearchService.php

```diff
@@ -70,7 +75,7 @@ function searchUserAction(KalturaESearchUserParams $searchParams, KalturaPager $
 	public function entryExportToCsvAction (KalturaMediaEsearchExportToCsvJobData $data)
 	{
 		if(!$data->userName || !$data->userMail)
-			throw new KalturaAPIException(APIErrors::USER_EMAIL_NOT_FOUND, $kuser);
+			throw new KalturaAPIException(APIErrors::USER_EMAIL_NOT_FOUND);
```

Comment:
USER_EMAIL_NOT_FOUND expect @USER_ID@ if we don't have $kuser maybe we should use different exception messages